### PR TITLE
WD-3412 - Add Real-time Ubuntu Beta optimised for Intel SoCs

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Added the new "Add Real-time Ubuntu Beta optimised for Intel SoCs" content
- Updated the download links and some content

## QA
- Open the demo and go to /download/iot/intel-iot
- Ensure the copy matches [the copydoc](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit#)

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-3412

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
@juanruitina how do you feel about partial  redesigns like this? Should we stick to the original layout? I feel mixing things like this might backfire, as if something doesn't look good, it might make people think the rebranding is going wrong.

**Comment by juanruitina** [comment]
I would feel inclined to stick to the scope of the request for now. Copy update looks fine UX wise, we can look at the whole page whenever we redesign it later in the cycle.

**Comment by lyubomir-popov** [comment]
I think that's a good call, @anthonydillon can you please switch to the original layout for that section? One that matches the adjacent ones.

**Comment by anthonydillon** [comment]
You got it @lyubomir-popov. Old styling applied.

File: templates/download/iot/intel-iot.html [hunk 1]
```diff
@@ -93,6 +93,25 @@ <h2 class="p-heading--3">Backed by Intel's partner ecosystem</h2>
...
+      <p><a href="https://register.gotowebinar.com/register/8712572348523284318?source=Canonical">Watch the joint webinar by Canonical and Intel on how real-time Ubuntu on Intel accelerates industrial transformation</a></p>
+      <p>Real-time Ubuntu on Intel SoCs is available via <a href="/pro">Ubuntu Pro</a>, Canonical's enterprise security and compliance subscription, covering all aspects of open infrastructure. With an <a href="/pro/dashboard">Ubuntu Pro subscription</a>, launching the kernel is as easy as:</p>
+      <div class="p-code-snippet">
+        <pre class="p-code-snippet__block--icon"><code>pro attach &lt;token&gt;</code></pre>
+        <pre class="p-code-snippet__block--icon"><code>pro enable realtime-kernel --access-only</code></pre>
+        <pre class="p-code-snippet__block--icon"><code>apt install ubuntu-intel-iot-realtime</code></pre>
+      </div>
+      <p>Are you interested in running real-time Ubuntu on Intel SoCs?</p>
+      <p><a href="/kernel/real-time/contact-us" class="p-button--positive">Get in touch</a></p>
```
Comment (petesfrench) [inline templates/download/iot/intel-iot.html:110]:
```suggestion
      <p><a href="/kernel/real-time/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a></p>
```

**Review by petesfrench — APPROVED** [review]
LGTM, just one code change

**Comment by anthonydillon** [comment]
We are +1 from PdM on Jira

File: templates/download/iot/intel-iot.html (hunk 1 above)
Comment (anthonydillon) [reply templates/download/iot/intel-iot.html:110]: I don't want to do that because that posts the data to a different formID

**Pull request merged**